doc/stdlib/libc.rst

```diff
@@ -40,7 +40,10 @@
 
 .. function:: time(t::TmStruct)
 
-   Converts a ``TmStruct`` struct to a number of seconds since the epoch.
```

Comment:
> a literal array of objects?

That syntax has worked quite nicely in Docile &ndash; especially for documenting functions with mutating and non-mutating versions:

```jl
"..."
[foo, foo!]
```

---

@one-more-minute would it be possible to split ``base/docs/helpdb.jl`` into several files, perhaps with the same structure as ``docs/stdlib/*``? Might help highlight anything else that's missing/duplicated more clearly.